scripts/scil_compute_bundle_voxel_label_map.py

```diff
@@ -49,10 +49,10 @@ def _build_arg_parser():
     p.add_argument('--nb_pts', type=int,
                    help='Number of divisions for the bundles.\n'
                         'Default is the number of points of the centroid.')
-    p.add_argument('--min_streamline_count', type=int, default=100,
+    p.add_argument('--min_streamline_count', type=int, default=100000,
```

Comment:
That's a question for @mdesco , I see now that virtually everyone using scil_compute_bundle_voxel_label_map.py does not save the TRK (optional, but internally the TRK has been trimmed), but then use the voxel labels map with the original TRK (not trimmed) and there is a discrepancy.

This trimming seems to increase risk of error, since most people don't know its happening. This could be problematic when used outside of tractometry flow. So what do we do?

Reply:
Ok I think I get it. But if we really want to make sure we don't trim anything shouldn't we set the default to `None` instead of `100000`?

Reply:
@frheault I agree with your EDIT idea above. 

Also missing space between cutting and operations